packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_map_renderer_android.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+/// Android map renderer
+enum AndroidMapRenderer {
+  /// Latest renderer type.
+  latestRenderer,
```

Comment:
"Renderer" is redundant in the context of an enum called `AndroidMapRenderer`.

Reply:
This is because [`default`](https://developers.google.com/android/reference/com/google/android/gms/maps/MapsInitializer#initialize(android.content.Context,%20com.google.android.gms.maps.MapsInitializer.Renderer,%20com.google.android.gms.maps.OnMapsSdkInitializedCallback):~:text=is%20provided%2C%20the-,default,-preference%20is%20taken) can't be used as an identifier because it's a keyword, so all values are suffixed.
I'll change `default` to `platformDefault` and that should do the trick.

Reply:
`Renderer` suffix was used because 'default' can't be used as an identifier because it's a keyword.
Default is now named as `platformDefault` as same naming is used [here](https://github.com/flutter/plugins/blob/c3955d221178056351f05f33d042b3b96b08b9a3/packages/url_launcher/url_launcher_platform_interface/lib/src/types.dart#L14).